### DISCUSSION TITLE
CUSA24705 Horizon Forbidden West (EU)

### DISCUSSION THREAD

**Discussion by iioriyagami** [GoldHEN/GoldHEN_Cheat_Repository#54] [Cheat requests]
Can we please get a infinite resources/max resources on use cheat for CUSA24705 , Horizon Forbidden West please. 
Thank you.

**Comment by kmeps4** [comment]
Hello there:

On GoldHEN side we do not create/develop cheats... if you need to ask an exclusive cheat, please redirect to: https://twitter.com/PS4Trainer and join their Discord Server and ask for needed Cheat.

If you wanna Submit a PS2/PS4 cheat you have created, please go to GoldHEN discord: https://discord.gg/pwfKbWzEx3 and submit them on the respective Cheat channel.

Have a great one!!!

**Reply by iioriyagami** [reply]
Hey man. Thanks for the reply and I apologise for posting here. I learned how to make cheats myself so it's all cool. Thank you. Have a good day/night

**Comment by Vuhivuhi** [comment]
Hello  iioriyagami
Can you teach me how to create cheats please.
Thank you so much.

**Comment by iioriyagami** [comment]
Hey I myself am stuck on making cheat codes.
I’m trying once I figure it out I will let you know.
Thanks
Have a good day

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Vuhivuhi** [comment]
Hello bros,
Can you help me to make cheat code for "front mission 2 remark ver 1.01"
please.
Many thanks bro.
